src/components/content/monitor/Monitor.tsx

```diff
@@ -0,0 +1,326 @@
+                const serviceNameList: { value: string; label: string }[] = [];
+                const cusServiceNameList: { value: string; label: string }[] = [];
+                if (rsp.length > 0) {
+                    rsp.forEach((serviceVo: ServiceVo) => {
+                        const cusServiceName: { value: string; label: string } = {
+                            value: serviceVo.customerServiceName ?? '',
+                            label: serviceVo.customerServiceName ?? '',
+                        };
+                        cusServiceNameList.push(cusServiceName);
+                        const serviceName: { value: string; label: string } = {
+                            value: serviceVo.name,
+                            label: serviceVo.name,
+                        };
+
+                        serviceNameList.push(serviceName);
+                    });
+                    setDeployedServiceList(rsp);
+                    setServiceNameList(serviceNameList);
+                    setCustomerServiceNameList(cusServiceNameList);
+                }
+            })
+            .catch((error: Error) => {
+                setDeployedServiceList([]);
+                setServiceNameList([]);
+                setCustomerServiceNameList([]);
+                Tip('error', 'fetch all deployed services error.');
+                setQueryResult(true);
+                console.error('fetch all deployed services error: ', error);
+            });
+    }, []);
+
+    const handleChangeServiceName = (selectServiceName: string) => {
+        if (selectServiceName) {
+            const cusServiceNameList: { value: string; label: string }[] = [];
+            deployedServiceList.forEach((serviceVo: ServiceVo) => {
+                if (serviceVo.name === selectServiceName) {
+                    const cusServiceName: { value: string; label: string } = {
+                        value: serviceVo.customerServiceName ?? '',
+                        label: serviceVo.customerServiceName ?? '',
+                    };
+                    cusServiceNameList.push(cusServiceName);
+                }
+            });
+            setCustomerServiceNameList(cusServiceNameList);
+        }
+    };
+
+    const handleChangeCustomerServiceName = (selectCustomerServiceName: string) => {
+        form.setFieldsValue({ ServiceId: getCurrentServiceId(selectCustomerServiceName) });
+    };
+
+    const getCurrentServiceId = (selectCustomerServiceName: string): string => {
+        let currentServiceID: string = '';
+        if (selectCustomerServiceName) {
+            deployedServiceList.forEach((serviceVo) => {
+                if (selectCustomerServiceName === serviceVo.customerServiceName) {
+                    currentServiceID = serviceVo.id;
+                }
+            });
+        }
+        return currentServiceID;
+    };
+
+    const layout = {
+        labelCol: { span: 8 },
+        wrapperCol: { span: 16 },
+    };
+
+    const tailLayout = {
```

Comment:
Do we need this additional wrapping of `wrapperCol` object in `tailLayout` and again use spread operator.

Reply:
fixed.